flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/functions/SystemFunctionUtils.java

```diff
@@ -108,83 +108,305 @@ public static TimestampData toTimestamp(String str, String format, String timezo
         }
     }
 
-    public static int timestampDiff(
-            String symbol,
+    public static Integer timestampDiff(
+            String timeIntervalUnit,
             LocalZonedTimestampData fromTimestamp,
             LocalZonedTimestampData toTimestamp) {
+        if (fromTimestamp == null || toTimestamp == null) {
+            return null;
+        }
         return timestampDiff(
-                symbol, fromTimestamp.getEpochMillisecond(), toTimestamp.getEpochMillisecond());
+                timeIntervalUnit,
+                fromTimestamp.getEpochMillisecond(),
+                ZoneId.systemDefault().getId(),
+                toTimestamp.getEpochMillisecond(),
+                ZoneId.systemDefault().getId());
```

Comment:
I think it's potentially risky to use `ZoneId.systemDefault` by default since crossing Daylight Saving Time boundaries might cause unexpected results. Why not just stick to UTC and remove the "TimeZone" parameter from all `timestampdiff` function (they're not exposed to user anyway)?